src/isSameDay/index.ts

```diff
@@ -23,11 +23,11 @@ import requiredArgs from '../_lib/requiredArgs/index'
  * var result = isSameDay(new Date(2014, 8, 4, 6, 0), new Date(2014, 8, 4, 18, 0))
  * //=> true
  */
-export default function isSameDay(dirtyDateLeft, dirtyDateRight) {
+export default function isSameDay(dirtyDateLeft: Date | number, dirtyDateRight: Date | number) {
```

Comment:
Thank you for the PR 👍 
Let's return type at function

```suggestion
export default function isSameDay(dirtyDateLeft: Date | number, dirtyDateRight: Date | number): boolean {
```